app/helpers/axios.js

```diff
@@ -0,0 +1,12 @@
+const axios = require('axios');
+
+const logger = require('../logger');
+const { badGatewayError } = require('../errors');
+
+exports.getWeetRandom = (method, url, data = '') =>
```

Comment:
We can rename this method to be more generic and can be used for requesting whatever external service. Maybe something like `request`

Reply:
I agree. Thanks